data/hails.txt

```diff
@@ -3056,3 +3056,23 @@ phrase "tribute battle message"
 phrase "tribute surrendered message"
 	word
 		"We surrender. We will pay you <tribute> per day to leave us alone."
+		
+phrase "pirate tribute ignored"
+	word
+		"We're not going to waste our time on a nobody like you."
+		
+phrase "pirate tribute battle"
+	word
+		"<last>, you have taunted us for the last time! Prepare to die!"
```

Comment:
To be fair, if the player has never taunted pirates before, then he/she has only done it once, and first = last, in that case.